pkg/mdns/controller.go

```diff
@@ -47,10 +48,11 @@ func (c *MicroShiftmDNSController) Run(ctx context.Context, ready chan<- struct{
 
 	ifs, _ := net.Interfaces()
 
+	excludedInterfacesRegexp := regexp.MustCompile("^[A-Fa-f0-9]{15}|ovn.*|br-int|veth.*|cni.*|ovs-system$")
```

Comment:
Can we add a comment with examples of interfaces names matching the regex?

Reply:
I was thinking to match on the driver of interface, but current code changes less and is more clean :-)

`^[A-Fa-f0-9]{15}` matches the ovs container interface (15 bits of the container sandbox ID)
`ovn.*` matches the ovn management port `ovn-k8s-mp0`
`br-int` matches the ovnk integration bridge
`cni.*` matches cni0
`ovs-system$` matches the ovs system interface.

Reply:
btw, do we listen on both br-ex and physical interface (like eth0)?
eth0 is added as ovs port of br-ex.

Reply:
> > btw, do we listen on both br-ex and physical interface (like eth0)? eth0 is added as ovs port of br-ex.
> 
> Right, it will listen on both. But I don't think we have a simple way of filtering the interface connected to br-ex without digging into openvswitch database, and I'm not sure if it's safe to only listen on "eth0" and exclude br-ex. Do you want me to try that?
if bonding is used on the OVS side that could fail to work.

I tend to think that duplicated replies is better than no replies.

Reply:
Can we add this list to a comment in the code, explaining the logic of filtering?

Reply:
Oh of course, that makes sense